### DISCUSSION TITLE
CodeTogether Plugin

### DISCUSSION THREAD

**Discussion by rrangelo** [apache/netbeans#5788] [Ideas]
Hello! I am a Netbeans user for 15 years, I would like to be able to use [CodeTogether](https://www.codetogether.com/live/) within the IDE, it would be great to be able to have this functionality.

**Comment by Chris2011** [comment]
I like the idea about this only thing that it costs in some cases. Anyway, it is also on my todo list to create a plugin for NetBeans users and I also thought about Cross IDE support but that is out of scope. So maybe it would be better to ask the developers of CodeTogether to create a plugin for NetBeans. They already have the code for the IntelliJ and Eclipse Plugin so java based and I hope that they have a good codebase to just need to change the UI part which is NetBeans specific.

**Reply by Chris2011** [reply]
But this is just my ideal thinking, we all know that real world can looks different.

**Comment by Chris2011** [comment]
I created a ticket at https://github.com/Genuitec/CodeTogether/issues/405
